### PR TITLE
Fixes #129 (setting of img freq)

### LINKED ISSUE #129: Tangled setting of img[].trans, img[].freq
As noted by @ParfenovS in comments for #118, the way the image frequency is defined is incorrect in some cases, which would muck up the image if there were more than 1 species.

### PR DESCRIPTION
**Pull request by allegroLeiden**
Fix #129

### CODE REVIEW

File: src/aux.c [hunk 1]
```diff
@@ -90,6 +90,7 @@ parseInput(inputPars *par, image **img, molData **m){
     (*img)[i].nchan=0;
     (*img)[i].velres=-1.;
     (*img)[i].trans=-1;
+    (*img)[i].molI=-1;
```
Comment (smaret) [inline src/aux.c]: Could you please document this new optional parameter in the user manual?

**Comment by smaret** [comment]
Except for missing documentation, this PR looks good to me.

**Comment by smaret** [comment]
Now that I've merged #128, could you please merge master into this one?

**Comment by allegroLeiden** [comment]
You got it.

**Pull request merged**